riff-raff/app/persistence/TargetDynamoRepository.scala

```diff
@@ -42,7 +40,7 @@ class TargetDynamoRepository(config: Config) extends DynamoRepository(config) {
       target: Target,
       projectName: String,
       lastSeen: DateTime
-  ): Option[Either[DynamoReadError, TargetId]] =
+  ): Unit =
     exec(table.put(TargetId(target, projectName, lastSeen)))
```

Comment:
`Table.put()` now returns `Unit` rather than the old value- you can use `Table.putAndReturn()` if you want a value returned, which allows you to specify whether it's the prior value or the new one you want (due to https://github.com/scanamo/scanamo/pull/486 - release with [1.0-M13](https://github.com/scanamo/scanamo/releases/tag/v1.0-M13))

Reply:
I presume this will throw an exception if the update fails, but this seems safe given that our only usage of this function is here:

https://github.com/guardian/riff-raff/blob/295ec4b91f9ee1769bc5daf3f46da59f1471bf4f/riff-raff/app/ci/TargetResolver.scala#L57-L59

Reply:
Yeah, I think it would always have thrown an exception if the update fails, the difference is just that if the _update_ succeeds, there's now no value being returned, and no chance of a `DynamoReadError` representing a deserialisation error. And you're right, `TargetResolver` is catching any erxception!